x-pack/plugins/infra/server/lib/alerting/metric_threshold/lib/metric_query.ts

```diff
@@ -77,8 +75,8 @@ export const getElasticsearchMetricQuery = (
             date_range: {
               field: timefield,
               ranges: Array.from(Array(Math.floor((to - from) / intervalAsMS)), (_, i) => ({
-                from: from + intervalAsMS * i + offsetInMS,
-                to: from + intervalAsMS * (i + 1) + offsetInMS,
```

Comment:
I imagine to solve the "delivery delay" situation we can just use a hard-coded value here of -60000 to shift back the range by 1m to avoid that final minute where things might be "too volatile" re: ES indexing still completing?

Reply:
Yep, that makes sense 👍

Reply:
👍 on this. I don't want to see a regression on delivery delays, so let's hard code shaving off the past 1 minute.

Reply:
Added that in.